pkg/utils/etcdutil/etcdutil.go

```diff
@@ -88,7 +89,7 @@ func CheckClusterID(localClusterID types.ID, um types.URLsMap, tlsConfig *tls.Co
 		trp := &http.Transport{
 			TLSClientConfig: tlsConfig,
 		}
-		remoteCluster, gerr := etcdserver.GetClusterFromRemotePeers(nil, []string{u}, trp, true)
```

Comment:
Is the default value of the new function `GetClusterFromRemotePeers` true?

Reply:
It is introduced by #8017 and https://github.com/etcd-io/etcd/pull/17330, it means `Enable 3.4 to be compatible with next version 3.5, to allow 3.4 server to join 3.5 cluster and start on 3.5 schema.`